engine-plugins/identity-ldap/src/main/java/org/camunda/bpm/identity/impl/ldap/LdapUserQueryImpl.java

```diff
@@ -63,7 +67,16 @@ protected LdapIdentityProviderSession getLdapIdentityProvider(CommandContext com
 
   @Override
   public UserQuery desc() {
-    throw new UnsupportedOperationException("The LDAP identity provider does not support descending search order.");
+    // provide this exception then a popup will be visible in the admin task, but display will run correctly
+    if (ldapConfiguration != null && !ldapConfiguration.sortControlSupported)
+      throw new UnsupportedOperationException("The LDAP identity provider does not support descending search order.");
+
+    // assign a default order descending if the list is empty
+    if (getOrderingProperties() != null && getOrderingProperties().isEmpty()) {
+      QueryOrderingProperty orderingProperty = new QueryOrderingProperty(UserQueryProperty.USER_ID, Direction.DESCENDING);
+      this.setOrderingProperties(Collections.singletonList(orderingProperty));
+    }
```

Comment:
```suggestion
```
We don't need to define a default ordering property when no ordering properties exist since the REST API has a validation anyways and throws an error in case `sortBy` hasn't been specified.

Cf. https://github.com/camunda/camunda-bpm-platform/blob/483728/webapps/src/main/java/org/camunda/bpm/cockpit/rest/dto/AbstractRestQueryParametersDto.java#L73